minicluster/src/main/java/tachyon/master/AbstractLocalTachyonCluster.java

```diff
@@ -45,6 +47,7 @@
 /**
  * Local Tachyon cluster.
  */
+@NotThreadSafe
```

Comment:
I would like to, but I don't see a style rule at http://checkstyle.sourceforge.net/ that we could use for that.